contrib/jenkins_tests/globals.sh

```diff
@@ -236,7 +203,43 @@ function do_check_result()
 #
 function do_get_ip()
 {
-    for ip in $(ibdev2netdev | grep Up | grep "$2" | cut -f 5 -d ' '); do
+    sv_ifs=${IFS}
+    netdevs=$(ibdev2netdev | grep Up | grep "$2" | cut -f 5 -d ' ')
+    IFS=$'\n' read -rd '' -a netdev_ifs <<< "${netdevs}"
+    lnkifs=$(ip -o link | awk '{print $2,$(NF-2)}')
+    IFS=$'\n' read -rd '' -a lnk_ifs <<< "${lnkifs}"
+    IFS=${sv_ifs}
+    ifs_array=()
+
+    for nd_if in "${netdev_ifs[@]}" ; do
+        found_if=''
+        for v_if in "${lnk_ifs[@]}" ; do
+            if [ ! -z "$(echo ${v_if} | grep ${nd_if})" ] ; then
+                mac=$(echo "${v_if}"| awk '{ print $NF }') #; echo "mac=$mac"
```

Comment:
Stray debug message? Another in this function as well.

Reply:
what do you mean?

Reply:
I'm referring to comments `#; echo "mac=$mac"` and `#-#echo "${nd_if} --> ${if_name::-1} "`. If these were only for debugging, my suggestion is to delete them.